sym/ed551a6a-f80e-4700-861b-a037225be664/symbol.lp

```diff
@@ -0,0 +1,96 @@
+(librepcb_symbol ed551a6a-f80e-4700-861b-a037225be664
+ (name "TCA9548A")
+ (description "Low-Voltage 8-Channel I2C Switch with Reset")
+ (keywords "i2c,mux,multiplexer")
+ (author "Marc Wittwer")
+ (version "0.1")
+ (created 2019-05-31T11:16:46Z)
+ (deprecated false)
+ (category e96d0dda-3100-4e68-8c33-e2a5bc80428b)
```

Comment:
This is the wrong category. I'm not sure what the correct one would be, maybe "Integrated Circuits > Interface > Analog Switches, Multiplexers, Demultiplexers"? @rnestler @ubruhin

Reply:
I agree, since it is a multiplexer and definitely not a |Single board computer" :slightly_smiling_face: